.github/workflows/test.yml

```diff
@@ -7,37 +7,33 @@ on:
   pull_request:
 
 # Current configuration:
-# - django 2.2, python 3.6
-# - django 3.0, python 3.7
-# - django 3.1, python 3.8
-# - django 3.2, python 3.9
-# - django 4.0, python 3.10
-# - django 4.1, python 3.10
+# - django 3.2, python 3.7
```

Comment:
suggestion: drop 3.7 as well since all supports ends in a couple of weeks - https://endoflife.date/python

And don't forget to update https://github.com/wagtail/telepath/blob/main/setup.py#L21